cibyl/plugins/openstack/features/network.py

```diff
@@ -0,0 +1,54 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+
+from cibyl.cli.argument import Argument
+from cibyl.plugins.openstack.features import OpenstackFeatureTemplate
+
+LOG = logging.getLogger(__name__)
+
+
+class HA(OpenstackFeatureTemplate):
+    """Feature representing high availability in an Openstack deployment."""
```

Comment:
I would change the content of the class docstring to something we can display to the uesr when he runs `cibyl --features'

The output will look like this:

```
Network:

  * HA - Highly available OpenStack deployment with at least 2 controllers
  * ...
```

And then we can perhaps use "Highly available OpenStack deployment with at least 2 controllers" from the class docstring

Reply:
That's a neat idea, I'll incorporate it